cmd/kpod/pull.go

```diff
@@ -1,31 +1,33 @@
 package main
 
 import (
+	"fmt"
 	"os"
 
-	"fmt"
+	"golang.org/x/crypto/ssh/terminal"
 
-	"github.com/containers/image/docker/reference"
-	"github.com/containers/image/pkg/sysregistries"
-	"github.com/containers/image/transports/alltransports"
 	"github.com/containers/image/types"
+	"github.com/kubernetes-incubator/cri-o/libpod"
+	"github.com/kubernetes-incubator/cri-o/libpod/common"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
 var (
 	pullFlags = []cli.Flag{
-		cli.BoolFlag{
-			// all-tags is hidden since it has not been implemented yet
-			Name:   "all-tags, a",
-			Hidden: true,
-			Usage:  "Download all tagged images in the repository",
-		},
 		cli.StringFlag{
 			Name:  "signature-policy",
 			Usage: "`pathname` of signature policy file (not usually used)",
 		},
+		cli.StringFlag{
+			Name:  "authfile",
+			Usage: "Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
+		},
+		cli.StringFlag{
+			Name:  "creds",
+			Usage: "`credentials` (USERNAME:PASSWORD) to use for authenticating to a registry",
+		},
```

Comment:
Personally I would have preferred seeing this PR be merged, then add the creds in later in  a separate PR (merge small and often).  But if you're doing --creds, we probably ought to also do the --cert-dir param as well.

Reply:
and --tls-verify